Streetcode/Streetcode.XIntegrationTest/ControllerTests/Streetcode/Create/StreetcodeCreateControllerTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Streetcode.DAL.Entities.Streetcode;
+using Streetcode.XIntegrationTest.ControllerTests.Utils;
+using Streetcode.XIntegrationTest.ControllerTests.Utils.BeforeAndAfterTestAtribute.Streetcode;
+using System.Net;
+using Xunit;
+
+namespace Streetcode.XIntegrationTest.ControllerTests.Streetcode.Create
+{
+    public class StreetcodeCreateControllerTests : BaseControllerTests, IClassFixture<CustomWebApplicationFactory<Program>>
+    {
+        public StreetcodeCreateControllerTests(CustomWebApplicationFactory<Program> factory)
+           : base(factory, "/api/Streetcode")
+        {
+
+        }
+
+        [Fact]
+        [ExtractCreateTestStreetcode]
+        public async Task Create_ReturnsSuccessStatusCode()
+        {
+            // Arrange
+            var streetcodeCreateDTO = ExtractCreateTestStreetcode.StreetcodeForTest;
+
+            // Act
+            var response = await client.CreateAsync(streetcodeCreateDTO);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        }
+
+
+        [Fact]
+        [ExtractCreateTestStreetcode]
+        public async Task Create_CreatesNewStreetcode()
+        {
+            // Arrange
+            var streetcodeCreateDTO = ExtractCreateTestStreetcode.StreetcodeForTest;
+
+            // Act
+            var response = await client.CreateAsync(streetcodeCreateDTO);
+            var streetcodeId = StreetcodeIndexFetch.GetStreetcodeByIndex(streetcodeCreateDTO.Index);
+            var getResponse = await client.GetByIdAsync(streetcodeId);
+            var fetchedStreetcode = CaseIsensitiveJsonDeserializer.Deserialize<StreetcodeContent>(getResponse.Content);
+
+            // Assert
+            // Check if the fetched Streetcode matches the one we created
```

Comment:
Please remove comment. It's all written in assert, and we can understand what it checking

Reply:
done